odoo/addons/base/data/res_company_demo.xml

```diff
@@ -4,5 +4,7 @@
     <record id="main_company" model="res.company">
         <field name="name">ProBike Inc</field>
     </record>
+
+    <function model="ir.config_parameter" name="set_param" eval="('i18n.default.server', 'https://odoo-dev.github.io')"/>
```

Comment:
@mart-e could you tell me very short what idea you guys have with this? :D

Reply:
@Yenthe666 not  _you guys_, only me making a PoC. Don't take what you see here as granted, we have **not** yet decided on the architecture to use. I can tell you that it is very likely the github.io website won't be present in the final dev.